CMakeLists.txt

```diff
@@ -50,7 +50,7 @@ option(WITH_DNF5DAEMON_TESTS "Build dnf5daemon tests (requires configured dbus)"
 option(WITH_SANITIZERS "Build with address, leak and undefined sanitizers (DEBUG ONLY)" OFF)
 
 # build options - bindings
-option(WITH_GO "Build Go bindings" OFF)
+option(WITH_GO "Build Go bindings" ON)
```

Comment:
I think we probably don't want to turn it on by default just yet.

I have spent some time trying it out and I don't think the bindings actually work.
Were you able to use them?

I would like to set up some tests the way we have for Python or Perl before we enabled them.